packages/components/src/Skeleton/Skeleton.component.js

```diff
@@ -46,7 +46,16 @@ function getTranslatedType(t, type) {
  * @param {number} props.height height to override size's height
  * @param {string} props.className classes to apply on skeleton
  */
-function Skeleton({ heartbeat, type, size, width, height, name, className, t }) {
+function Skeleton({
```

Comment:
This will impact a bunch of components.
What have I done on skeleton:
- useTranslation instead of withTranslation HOC
- move the default props to default values

Why have I done that:
- (jest/enzyme) withTranslation was snapshot with `<withI18nTranslation(componentName)>`. But not anymore, the component name is `Component.displayName` which is not provided in a lot of components.
- (jest/enzyme) the default props and the HOC injected `t` appears in snapshots now. I just didn't want that pollution and avoid the fact that future change in skeleton will impact tons of snapshots

I surely won't do that on all the components, but here the snapshot are cleaner with a simple `<Skeleton>`